roxie/ccd/ccdcontext.cpp

```diff
@@ -1775,8 +1775,9 @@ class CRoxieContextBase : implements IRoxieAgentContext, implements ICodeContext
     }
     virtual IEngineContext *queryEngineContext() { return NULL; }
     virtual char *getDaliServers() { throwUnexpected(); }
-    virtual unsigned getWorkflowId() { return 0; } // this is a virtual which is implemented in IGlobalContext
-
+    // The following method isn't used.  Renaming and leaving here to avoid changing interface
+    // (Note, this getWorkflowId doesn't implement virtual from any derived classess.)
+    virtual unsigned getWorkflowIdDeprecated() { throwUnexpected(); }
```

Comment:
eclhelper.hpp doesn't include jexcept.hpp so throwUnexpected can't be called at the moment from  IGlobalCodeContext.  If jexcept.hpp is included in eclhelper.hpp, then jexcept.hpp would need to be installed in the include directory.   I don't think I can even call "throw makeStringException" without also installing other jlib headers.  It could just return 0?

Reply:
fair point. I would leave it as it is.